datasets/gov/lsd/cl/ja_asmenys.csv

```diff
@@ -7,23 +7,23 @@ id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri
 ,,,,,,comment,type,,"update(type: ""ref"", ref: ""datasets/gov/lsd/cl/sektoriai/Sektorius"")",4,open,spinta:1476,,
-,,,,,reg_data,date,D,,,4,open,,"Įregistravimo data","Juridinio asmens įregistravimo data (JAR duomuo)"
+,,,,,iregistruotas,date,D,,,4,open,,"Įregistravimo data","Juridinio asmens įregistravimo data (JAR duomuo)"
 ,,,,,nforma,integer,,,,4,open,,"Nuosavybės formos","Nuosavybės formos"
 ,,,,,,enum,,,11,,,,"Kodas","Viešoji nuosavybė, kurioje valstybės ar savivaldybės nuosavybė sudaro 50% arba daugiau kaip 50 % subjekto įstatinio kapitalo ir jo sudėtyje nėra užsienio investuotojų kapitalo"
 ,,,,,,,,,12,,,,"Kodas","Viešoji nuosavybė, kurioje valstybės ar savivaldybės nuosavybė sudaro 50% arba daugiau kaip 50 % subjekto įstatinio kapitalo ir jo sudėtyje yra užsienio investuotojų kapitalo"
 ,,,,,,,,,21,,,,"Kodas","Privati nuosavybė, kurioje Lietuvos fiziniams bei juridiniams asmenims priklauso daugiau kaip 50 % subjekto įstatinio kapitalo ir jo sudėtyje nėra užsienio investuotojų kapitalo"
 ,,,,,,,,,22,,,,"Kodas","Privati nuosavybė, kurioje Lietuvos fiziniams bei juridiniams asmenims priklauso daugiau kaip 50 % subjekto įstatinio kapitalo ir jo sudėtyje yra užsienio investuotojų kapitalo"
 ,,,,,,,,,23,,,,"Kodas","Privati nuosavybė, kurioje užsienio fizinių bei juridinių asmenų nuosavybė sudaro daugiau kaip 50 % subjekto įstatinio kapitalo"
-,,,,,veik_poz,integer,,,,3,open,,"Veikiantys (reikšmė „1“ arba tuščias)","Šis požymis nurodo, kad tam tikram momentui, statistinis vienetas yra veikiantis, t.y. vykdo ekonominę veiklą, turi darbuotojų ir (arba) pajamų. Veikiančių statistinių vienetų sąrašas atnaujinamas vieną kartą per metus ir fiksuojamas sausio 1 d."
+,,,,,veikiantis,integer,,,,3,open,,"Veikiantys (reikšmė „1“ arba tuščias)","Šis požymis nurodo, kad tam tikram momentui, statistinis vienetas yra veikiantis, t.y. vykdo ekonominę veiklą, turi darbuotojų ir (arba) pajamų. Veikiančių statistinių vienetų sąrašas atnaujinamas vieną kartą per metus ir fiksuojamas sausio 1 d."
```

Comment:
Iš kur šis ateina šis duomuo - iš JAR? Ar tai yra išvestinis duomuo iš JAR duomenų?